### PR TITLE
Sound: Fix Faux Sound & Add TPA6165 headphone amp gain

### PR DESCRIPTION
**Pull request by Kamin4ri**
My previous pull request, in only one commit :p

Signed-off-by: Kamin4ri <kamin4ridev@gmail.com>

### CODE REVIEW

File: app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java [hunk 1]
```diff
@@ -119,11 +126,42 @@ public static boolean hasHeadphonePowerAmpGain() {
...
+    public static void setHeadphoneTpaGain(String value, Context context) {
+        // Headphone Amp Gain is register 0x7.
+        // Zero corresponds to 185 (0xb9). The min value is -24 (0xa1) and max is 6 (0xbf).            
+        int gain = 185 + Utils.strToInt(value);
+        run(Control.chmod("222", TPA6165_SET_REG), TPA6165_SET_REG, context);
+        run(Control.write("0x07 0x" + Integer.toHexString(gain), TPA6165_SET_REG), TPA6165_SET_REG, context);
+    }
+
+    public static String getHeadphoneTpaGain() {
+        String strGain = RootUtils.runCommand("cat " + TPA6165_REGISTERS_LIST + " | awk \"/0x7/\" | cut -c9-13");
```
Comment (Grarak) [inline app/src/main/java/com/grarak/kerneladiutor/utils/kernel/sound/Sound.java]: about awk and cut. Possible to do that in java?

**Comment by Kamin4ri** [comment]
Yes. Worked just fine for me. You just gotta call it via RootUtils.runCommand() as I did there. I didnt know of any other way to get what i wanted, except these commands.

When doing this on a shell:
```bash
root@device:/# cat /sys/kernel/debug/tpa6165/registers | awk "/0x7/" | cut -c9-13
```
A list of registers (and values) is displayed in the terminal. To get the headphone amp gain's register (0x7),    i sent it to awk, and then to get only the register and value, i used cut. I think that command uses something in SuperSU/libsuperuser to do the same thing but using an invisible shell and returning the result as a String. I only know it works :p 

So yeah, i guess you can use any command as long as it returns something that can be passed to the app as a String...

**Comment by Kamin4ri** [comment]
Damn. Hold on. These commits shouldn't be here

**Comment by Grarak** [comment]
git reset --hard HEAD~[number of commits to remove]
git push -f

**Comment by Kamin4ri** [comment]
Will have to create another request. Resetting the branch to the previous state didn't mess with this diff.

**Pull request closed without merging**